portfolio/src/main/webapp/index.html

```diff
@@ -20,11 +19,21 @@
         </div>
         <section id="home">
             <div>
-                <h1>I'm Dhruvi Sheth.</h1>
+                <h2>Hello! I'm</h2>
+                <h1>Dhruvi Sheth.</h1>
                 <h3>Student | Aspiring Software Engineer | Tech Enthusiast</h3>
-                <a href="#">Learn More</a>
+                <a href="#contact">Get In Touch</a>
             </div>
         </section> 
+        <section id="about">
+            <div>
+                <h2>About Me</h2>
+                <p> I am a junior studying Computer Science at Rutgers University - New Brunswick. I was born in India, raised in Canada, and currently live in the United States. My hobbies include cooking (especially experimenting with new recipes), traveling, and spending time with my family and friends. On campus, I am the Co-President of Rutgers Women in Information Technology and Informatics, a organization built on the foundation of supporting and providing opportunities to women in the technology industry. I also serve as a Computer Science Peer Tutor for introductory computer science classes like Data Structures. Additionally, I work as a Technology Intern at the Rutgers Office of Career Exploration and Success.</p>
```

Comment:
Can you add newlines in the text here to prevent it from going over 80 characters on any one line?